### ISSUE TITLE
Question/Feature: Environment variable for Cirrus CLI

### ISSUE THREAD

**Issue by dschier-wtd** [cirruslabs/cirrus-cli#480]
Hi,

I am using cirrus-cli together with cirrus-ci and wanted to have tasks running for the CI, but not for CLI.
Is there an environment variable to indicate, if I am currently in the CI or CLI? If not, I want to request the same.

Best regards.

**Comment by fkorotkov** [comment]
You can check `CIRRUS_ENVIRONMENT`. It's set to `"CLI"` by the CLI.

**Comment by dschier-wtd** [comment]
Ah, that's awesome. Thanks a lot :)

**Comment by dschier-wtd** [comment]
I tested the above a bit. Unfortunately, something seems to be odd.

The below example will run, even if on CLI. A simple "echo $CIRRUS_ENVIRONMENT" also states, CLI is set.

```yaml
devel_docker_builder:
  name: "Deliver Devel Image"
  # only_if: "$CIRRUS_BRANCH == $CIRRUS_DEFAULT_BRANCH && \
  #           $CIRRUS_TAG == '' && \
  #           $CIRRUS_ENVIRONMENT == ''"

  only_if: "$CIRRUS_ENVIRONMENT != 'CLI'"
  skip: "$CIRRUS_ENVIRONMENT == 'CLI'"

  depends_on:
    - "Test Image"
```

Even worse. The `skip` and `depends_on` also seems to be ignored.

PS: I am on v0.70.1

**Comment by edigaryev** [comment]
> I tested the above a bit. Unfortunately, something seems to be odd.

It is indeed odd, thanks for reporting. The fix is now available as a part of [Cirrus CLI 0.70.2](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.70.2), which will be available shortly.

**Comment by dschier-wtd** [comment]
Aaaand, it works like a charm. Thanks again. It's awesome how fast you get in touch and fix these things :)
